meson.build

```diff
@@ -74,18 +75,33 @@ if get_option('wayland') != 'false'
   endif
 endif
 
+# WIN32
+use_win32 = false
+if get_option('win32') != 'false'
```

Comment:
false will disable it explicitly, true will try to force enable it and fail if dependencies not found. 'auto' (which is also what you get if you do not specify the arguments in the meson commandline) will try to enable it if the dependencies are found. In other words, if you run "meson build/" on windows it will enable win32 if it finds libva-win32